pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
interesting this went down?

Maybe you need to update your local pnpm?

Reply:
nice catch! I used a lower version, and needed to update and that probably messed up with the lock file